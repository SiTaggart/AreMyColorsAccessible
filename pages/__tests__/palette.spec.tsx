/* eslint-env jest */

import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Palette from '../palette';

describe('Palette Page', () => {
  it('renders without crashing', () => {
    ReactDOM.render(<Palette />, document.createElement('div'));
    const paletteCmp = renderer.create(<Palette />).toJSON();
    expect(paletteCmp).toMatchSnapshot();
  });
});
