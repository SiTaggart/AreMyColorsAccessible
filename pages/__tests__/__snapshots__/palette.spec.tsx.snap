// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Palette Page renders without crashing 1`] = `
<main
  className="container container--palette"
>
  <div
    className="layout layout--full"
  >
    <form
      className="paletteForm"
      onSubmit={[Function]}
    >
      <div
        className="paletteForm-element"
      >
        <h1
          className="paletteForm-heading"
        >
          <label
            className="form-label form-label--large"
            htmlFor="palette-form-input"
          >
            Add the colours from your palette
          </label>
        </h1>
        <input
          autoComplete="off"
          className="form-input form-input--noSpacing"
          id="palette-form-input"
          name="paletteFormInput"
          placeholder="e.g. #ccc, #fff or blue"
          type="text"
        />
      </div>
    </form>
    <section
      className="colorMatrix"
    >
      <table
        className="colorMatrix-table"
      >
        <thead>
          <tr
            className="colorMatrix-tr"
          >
            <td
              style={
                Object {
                  "width": "6rem",
                }
              }
            />
          </tr>
        </thead>
        <tbody />
      </table>
    </section>
  </div>
  <footer
    className="footer"
  >
    <nav>
      <ul>
        <li>
          <a
            href="/"
            onClick={[Function]}
          >
            Home
          </a>
        </li>
        <li>
          <a
            href="/palette"
            onClick={[Function]}
          >
            Palette
          </a>
        </li>
        <li>
          <a
            href="/about"
            onClick={[Function]}
          >
            About
          </a>
        </li>
      </ul>
    </nav>
  </footer>
</main>
`;
